test/terra/reference/ref_1q_clifford.py

```diff
@@ -776,3 +776,124 @@ def pauli_gate_counts_deterministic(shots, hex_counts=True):
         targets.append({'101': shots})
         targets.append({'011': shots})
     return targets
+
+# ==========================================================================
+# I-gate
+# ==========================================================================
+
+def id_gate_circuits_deterministic(final_measure=True):
```

Comment:
What is the purpose for adding these test circuits? Isn't it enough to have a test for identity and delay in the standard gate tests?

Reply:
The purpose here is to test the stabilizer simulator for the identity and delay gates